jetstream/administration/naming.md

```diff
@@ -0,0 +1,19 @@
+# Stream, Consumer, and Account Names
+
+Stream, Consumer (durable name), and Account names are used in both the
+subject namespace used by JetStream and the filesystem backing JetStream
+persistence.  This means that when naming streams, consumers, and
+accounts, names must adhere to subject naming rules as well as being
+friendly to the file system.
+
+We recommend the following guideline for stream, consumer, and account names:
+
+* Alphanumeric values are recommended.
+* Spaces, tabs, period \(`.`\), greater than \(`>`\) or asterix \(`*`\) are prohibited.
+* Limit names to 255 characters or less.
+* Do not use reserved file names like NUL, LPT1, etc.
+* Be aware that some file systems are case insentitive so do not
```

Comment:
Wouldn't it be file systems that are case sensitive that would be the issue?

Reply:
Actually, the issue is that it's case insensitive, but I'll look to clarify.  There is another spot I need to update as well.

Reply:
I've updated this PR.  On case insensitive systems, one could create a stream "Foo" and "foo" and directories would collide on disk.